### ISSUE TITLE
Querying questions for applicants who applied but were not accepted returns empty arrays

### ISSUE THREAD

**Issue by AJLiu** [HackGT/registration#191]
Try running the following query
```
{
  users(n: 5, filter: {applied:true, accepted: false}) {
    questions(names: ["name", "school", "school-year", "major", "gender", "race", "first-hackathon", "tshirt-size", "github"]) {
        value
        values
      }
  }
}
```
the result will be
```
{
  "data": {
    "users": [
      {
        "questions": []
      },
      {
        "questions": []
      },
      {
        "questions": []
      },
      {
        "questions": []
      },
      {
        "questions": []
      }
    ]
  }
}
```
The cause is [this line of code](https://github.com/HackGT/registration/blob/ee750cf218e5110df325aca2bfccdbed67351ad6/server/routes/api/graphql.ts#L153), is there any reason why the question items for application data depends on acceptance and not application submission?

**Comment by AJLiu** [comment]
@kexin-zhang

**Comment by petschekr** [comment]
Closed in #192
